### ISSUE TITLE
Runtime error on Mac: Could not load the file 'Plugin.BLE'

### ISSUE THREAD

**Issue by janusw** [xabre/xamarin-bluetooth-le#491]
## Steps to reproduce

1. Create a Xamarin.Forms application including a Mac project.

2. Add Plugin.BLE version 2.1.1 to the shared Xamarin.Forms project.

3. Build and run.

## Expected behavior

The app should start up without errors.

## Actual behavior

The app crashes with the error: Could not load the file 'Plugin.BLE'

### Crashlog

```
System.IO.FileNotFoundException: Could not load the file 'Plugin.BLE'.
  at System.AppDomain.Load (System.Reflection.AssemblyName assemblyRef, System.Security.Policy.Evidence assemblySecurity) [0x00073] in /Library/Frameworks/Xamarin.Mac.framework/Versions/Current/src/Xamarin.Mac/mcs/class/corlib/System/AppDomain.cs:758
  at System.AppDomain.Load (System.Reflection.AssemblyName assemblyRef) [0x00000] in /Library/Frameworks/Xamarin.Mac.framework/Versions/Current/src/Xamarin.Mac/mcs/class/corlib/System/AppDomain.cs:724
  at at (wrapper remoting-invoke-with-check) System.AppDomain.Load(System.Reflection.AssemblyName)
  at System.Reflection.Assembly.Load (System.Reflection.AssemblyName assemblyRef) [0x00000] in /Library/Frameworks/Xamarin.Mac.framework/Versions/Current/src/Xamarin.Mac/mcs/class/corlib/System.Reflection/Assembly.cs:440
  at ObjCRuntime.Runtime.CollectReferencedAssemblies (System.Collections.Generic.List`1[T] assemblies, System.Reflection.Assembly assembly) [0x00085] in /Users/builder/azdo/_work/1/s/xamarin-macios/src/ObjCRuntime/Runtime.cs:545
  at ObjCRuntime.Runtime.CollectReferencedAssemblies (System.Collections.Generic.List`1[T] assemblies, System.Reflection.Assembly assembly) [0x00085] in /Users/builder/azdo/_work/1/s/xamarin-macios/src/ObjCRuntime/Runtime.cs:545
  at ObjCRuntime.Runtime.RegisterEntryAssembly (System.Reflection.Assembly entry_assembly) [0x00032] in /Users/builder/azdo/_work/1/s/xamarin-macios/src/ObjCRuntime/Runtime.cs:506
  at ObjCRuntime.Runtime.RegisterAssemblies () [0x00001] in /Users/builder/azdo/_work/1/s/xamarin-macios/src/ObjCRuntime/Runtime.cs:482
  at AppKit.NSApplication.Init () [0x00022] in /Users/builder/azdo/_work/1/s/xamarin-macios/src/AppKit/NSApplication.cs:56
  at WTD.Mac.MainClass.Main (System.String[] args) [0x00001] in /Users/janus/git/wtd.apps.dfs.de/WTD.Mac/Main.cs:9
```

## Configuration

**Version of the Plugin:** e.g. 2.1.1

**Platform:** MacOS Big Sur (11.2.3)

**Device:** MacBook Pro

**Comment by smsissuechecker** [comment]
Hi @janusw,

I'm the friendly issue checker.
Thanks for using the issue template :star2:
I appreciate it very much. I'm sure, the maintainers of this repository will answer, soon.

**Comment by janusw** [comment]
This issue looks very related to #430, and it might actually be fixed by #431 (I couldn't check this yet because there is no release which includes the fix).

**Comment by janusw** [comment]
After I managed to build a nuget package (with the fixes from #493), I could indeed verify that the runtime error reported here goes away with nupkg built from current master (9d4ec4d4f718971a1aac0ef8bf986b8bfa492de9) plus PR #493. Thus I conclude that it was in fact fixed by ce2dd84fe619d333f4e3538ca01aff15c268ad76 (#431).

IMHO this issue can be closed once a release is available on nuget.org which includes this fix.
